xdem/coreg/affine.py

```diff
@@ -580,6 +582,11 @@ def _fit_pts_func(
                 f"'dem_to_be_aligned' had {points['tba'].size} valid points."
             )
 
+        # If raster was reference, invert the matrix
+        # TODO: Move matrix/invert_matrix to affine module?
```

Comment:
to be addressed in this PR?

Reply:
In the end I think we can keep it in the base module for now. Moving to affine.py would require overridding several `Coreg` functions and moving a lot of things to avoid circular imports...